File: 0169-majority-element/0169-majority-element.java
class Solution {
    public int majorityElement(int[] nums) {
        int count=0,candidate=0;
        for(int a:nums){
            if(count==0){
                candidate=a;
                
            } if(candidate==a){
                count++;
            }else{
                count--;
            }
            
        }
 return candidate;     
}  
}

File: 0169-majority-element/0169-majority-element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        # count=num=0
        # for i in nums:
        #     if count==0:
        #         num=i
        #     if num==i:
        #         count+=1
        #     else:
        #         count-=1
        # return num
        return sorted(nums)[len(nums)//2] 

